### ISSUE TITLE
cant resize image

### ISSUE THREAD

**Issue by alicera** [AIZOOTech/FaceMaskDetection#67]
target_shape=(160, 160),

(1,2160,2)  VS (1, 11000, 2)

because the anchor is alwalys  11000

**Comment by AIZOOTech** [comment]
Yes, if you want to change the input size, you need to change it the corresponding feature_map_size.

```
# this is the input size with 260.
feature_map_sizes = [[33, 33], [17, 17], [9, 9], [5, 5], [3, 3]]
# if your input is 160
feature_map_sizes = [[20, 20], [10, 10], [5, 5], [3, 3], [2, 2]]
```

**Comment by alicera** [comment]
how to define feature_map_sizes with image size?

**Comment by AIZOOTech** [comment]
With different input size, the feature map size for locate is different. I am sorry we didn't edit our codes to adapt different input size. The main reason is that we train the model with input size 260.
